openmmtools/forces.py

```diff
@@ -726,12 +745,14 @@ class RadiallySymmetricBondRestraintForce(RadiallySymmetricRestraintForce,
     """
 
     def __init__(self, energy_function, restraint_parameters,
-                 restrained_atom_index1, restrained_atom_index2):
+                 restrained_atom_index1, restrained_atom_index2,
+                 controlling_parameter_name='lambda_restraints'):
         # Initialize CustomBondForce.
         energy_function = energy_function.replace('distance(g1,g2)', 'r')
-        energy_function = 'lambda_restraints * ' + energy_function
-        super(RadiallySymmetricBondRestraintForce, self).__init__(restraint_parameters,
-                 [restrained_atom_index1], [restrained_atom_index2], energy_function)
+        energy_function = controlling_parameter_name + ' * ' + energy_function
```

Comment:
So I just noticed a potential bug here. If the user passes in an `energy_function` which has additive terms and not wrapped in `( )`, this could lead to unintended behavior, which would NOT throw an error. Off the top of my head, the only practical thing I can think of is some kind of external field or shifted function, but this is also an easy enough fix it may be worth fixing now.

e.g.
```python
>>> energy_function = '(K/2)*r^2 + 5*r'
>>> controlling_parameter_name = 'X`
>>> controlling_parameter_name + ' * ' + energy_function
X * (K/2)*r^2 + 5*r
```
the `5*r` term won't be scaled.

Easy fix is just `controlling_parameter_name + ' * (' + energy_function + ')'`

Reply:
Good catch! Will change it.